src/main/java/com/firebolt/jdbc/client/account/FireboltAccountClient.java

```diff
@@ -42,8 +38,7 @@ public FireboltAccountClient(OkHttpClient httpClient, ObjectMapper objectMapper,
      * @param accessToken the access token
      * @return the account
      */
-    public FireboltAccountResponse getAccount(String host, String account, String accessToken)
-            throws FireboltException, IOException {
+    public FireboltAccountResponse getAccount(String host, String account, String accessToken) throws FireboltException, IOException {
         String uri = format(GET_ACCOUNT_ID_URI, host, account);
```

Comment:
There are a lot of linting changes in this PR, making it harder to review the relevant code. Can we add some sort of linter to both local and CI so there's no arbitrary style changes? 
It can be run on the whole codebase to fix those issues (as a separate PR ofc) so then everything afterwards will only contain relevant code changes.

Reply:
Generally I agree with you and I am sorry that I have reformatted code in this PR. Unfortunately I do not know how to avoid showing such lines in the github diff tool.

Reply:
As I suggested in the comment above - running a linter tool that automatically fixes those issues for the whole repo, commiting that and using the tool as a part of your workflow/IDE and in CI. This would ensure no unnecessary style changes.

Reply:
next time